[2024-12-19T13:09:44.045+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-19T13:09:44.050+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: PRCS_TEST_FRAMEWORK1.ExecuteNotebook manual__2024-12-19T13:05:59.817359+00:00 [queued]>
[2024-12-19T13:09:44.053+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: PRCS_TEST_FRAMEWORK1.ExecuteNotebook manual__2024-12-19T13:05:59.817359+00:00 [queued]>
[2024-12-19T13:09:44.053+0000] {taskinstance.py:2866} INFO - Starting attempt 3 of 8
[2024-12-19T13:09:44.056+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): ExecuteNotebook> on 2024-12-19 13:05:59.817359+00:00
[2024-12-19T13:09:44.059+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=11090) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-19T13:09:44.060+0000] {standard_task_runner.py:72} INFO - Started process 11101 to run task
[2024-12-19T13:09:44.060+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'PRCS_TEST_FRAMEWORK1', 'ExecuteNotebook', 'manual__2024-12-19T13:05:59.817359+00:00', '--job-id', '2316', '--raw', '--subdir', 'DAGS_FOLDER/script/PRCS_TEST_FRAMEWORK1.py', '--cfg-path', '/tmp/tmp0ngxc_xf']
[2024-12-19T13:09:44.061+0000] {standard_task_runner.py:105} INFO - Job 2316: Subtask ExecuteNotebook
[2024-12-19T13:09:44.077+0000] {task_command.py:467} INFO - Running <TaskInstance: PRCS_TEST_FRAMEWORK1.ExecuteNotebook manual__2024-12-19T13:05:59.817359+00:00 [running]> on host 60778319043b
[2024-12-19T13:09:44.113+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Nattapat-process' AIRFLOW_CTX_DAG_ID='PRCS_TEST_FRAMEWORK1' AIRFLOW_CTX_TASK_ID='ExecuteNotebook' AIRFLOW_CTX_EXECUTION_DATE='2024-12-19T13:05:59.817359+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-19T13:05:59.817359+00:00'
[2024-12-19T13:09:44.113+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-19T13:09:44.121+0000] {base.py:84} INFO - Retrieving connection 'postgres_localhost'
[2024-12-19T13:09:52.436+0000] {base.py:84} INFO - Retrieving connection 'postgres_localhost'
[2024-12-19T13:09:52.442+0000] {logging_mixin.py:190} INFO - Success Running : http://localhost:8888/notebooks/Log_output/PRCS_TEST_FRAMEWORK1/test1_executed.ipynb
[2024-12-19T13:09:52.442+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-12-19T13:09:52.447+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-19T13:09:52.447+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=PRCS_TEST_FRAMEWORK1, task_id=ExecuteNotebook, run_id=manual__2024-12-19T13:05:59.817359+00:00, execution_date=20241219T130559, start_date=20241219T130944, end_date=20241219T130952
[2024-12-19T13:09:52.459+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-19T13:09:52.464+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-19T13:09:52.464+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
